arch/arm/boot/dts/overlays/vc4-kms-v3d-overlay.dts

```diff
@@ -115,6 +115,20 @@
 		};
 	};
 
+	fragment@15 {
```

Comment:
This is the wrong overlay - you want vc4-kms-v3d-pi4.

Reply:
Ah...